generator/internal/rust/templates/crate/src/builders.rs.mustache

```diff
@@ -86,7 +86,6 @@ pub mod {{Codec.ModuleName}} {
         {{#IsPageable}}
 
         /// Streams the responses back.
-        #[cfg(feature = "unstable-stream")]
         pub async fn stream(self) -> gax::paginator::Paginator<{{OutputType.Codec.QualifiedName}}, gax::error::Error> {
```

Comment:
It seems like this function should be `paginator()` now, since it does not return a stream?   Or maybe we should have 2 functions: `by_page()` returns the paginator with the stable API, `by_item()` returns the thing with item-by-item.

Reply:
Hmm. Changed to `paginator()`. I think it is a stream, conceptually, so the old name was fine. But eh.

> Or maybe we should have 2 functions

I considered `stream_items()` and `stream_pages()`.

I think 99% of use cases will want to iterate over the items. Returning the `Paginator` requires the xtra call to `.items()`...

But I think I prefer that there is only one way for applications to write the code. There is less to keep track of mentally.

Reply:
I don't love that we are breaking folks with the name change. :shrug:. Do you care?

Reply:
I don't love making breaking changes.  I would like having the wrong name in the 1.0 release even less, and it is a 0.x release for a reason.